rpm/3.4/arangodb3e.spec.in

```diff
@@ -257,20 +260,26 @@ else
   fi
 fi
 
+exit 0
+
 ## -----------------------------------------------------------------------------
 ## --SECTION--                                                             preun
 ## -----------------------------------------------------------------------------
 
 %preun
-if rpm -q --quiet systemd ; then
-  systemctl stop arangodb3
-  rm %{_sysconfdir}/systemd/system/arangodb3.service
-  systemctl daemon-reload
-else
-  chkconfig --level 2345 arangodb3 off
-  rm %{_sysconfdir}/init.d/arangodb3
+if test "$1" -eq 0; then
+  if rpm -q --quiet systemd ; then
+    systemctl stop arangodb3
+    rm %{_sysconfdir}/systemd/system/arangodb3.service
+    systemctl daemon-reload
+  else
+    chkconfig --level 2345 arangodb3 off
+    rm %{_sysconfdir}/init.d/arangodb3
+  fi
+  rm %{_sysconfdir}/logrotate.d/arangodb3
 fi
-rm %{_sysconfdir}/logrotate.d/arangodb3
+
+exit 0
```

Comment:
shouldn't this rather be the else case? Or was this exit missing previously like in all the other cases?

Reply:
The exit was missing